packages/pods-core/src/builtin/HTMLPod.ts

```diff
@@ -27,25 +29,45 @@ export class HTMLPublishPod extends PublishPod<HTMLPublishPodConfig> {
           default: true,
           nullable: true,
         },
+        convertTagNotesToLinks: {
+          type: "boolean",
+          default: false,
+          nullable: true,
+        },
+        convertUserNotesToLinks: {
+          type: "boolean",
+          default: false,
+          nullable: true,
+        },
       },
     }) as JSONSchemaType<HTMLPublishPodConfig>;
   }
 
   async plant(opts: PublishPodPlantOpts): Promise<any> {
     const { config, engine, note } = opts;
-    const { fname, convertLinks = true } = config;
+    const {
+      fname,
+      convertLinks = true,
+      convertTagNotesToLinks = false,
+      convertUserNotesToLinks = false,
+    } = config;
     const { data: econfig } = await engine.getConfig();
+    const overrideConfig = { ...econfig! };
+
+    const wsConfig = ConfigUtils.getWorkspace(overrideConfig);
```

Comment:
super nitpick: the naming `wsConfig` is a bit confusing since we call vscode workspace configs wsConfig. Can we change it to `workspaceConfig` instead?